src/ctia/entity/vulnerability/mapping.clj

```diff
@@ -67,7 +67,7 @@
       :temporal_score em/float-type
       :environmental_score em/float-type
       :remediation_level em/token
-      :temporal_severity em/float-type
+      :temporal_severity em/token
```

Comment:
Wouldn't that force a complete migration?

Reply:
In theory yes, but our instance does not use this field. Its definition was wrong.

I checked with this ElasticSearch query:

```json
{
    "query": {
        "exists": {
            "field": "impact.cvss_v3.temporal_severity"
        }
    }
}
```